templates/generic/tag_multiselect.html

```diff
@@ -0,0 +1,115 @@
+<div x-data="dropdown()" x-init="loadOptions()">
+  <input name="values" type="hidden" x-bind:value="selectedValues()">
+
+  <div id="linkTagDiv" data-url="{% url 'annotations:link_tag' object_id=object_id %}"></div>
+  <div id="unlinkTagDiv" data-url="{% url 'annotations:unlink_tag' object_id=object_id %}"></div>
+
+  <div class="inline-block relative w-fit">
+    <div class="flex flex-col items-center relative">
+      <div x-on:click="open" class="w-full">
+        <div class="my-2 p-1 flex select select-bordered rounded">
+          <div class="flex flex-auto flex-wrap">
+              <!-- The tag names in the input -->
+            <template x-for="(option,index) in selected" :key="options[option].value">
+              <div class="flex justify-center items-center m-1 font-medium py-1 px-1 rounded bg-gray-100 hover:bg-gray-100">
+                <div class="text-xs font-normal leading-none max-w-full flex-initial" x-model="options[option]" x-text="options[option].text"></div>
+                <div class="flex flex-auto flex-row-reverse">
+                  <div x-on:click.stop="remove(index,option)">
+                    <svg class="fill-current h-4 w-4 " role="button" viewBox="0 0 20 20">
+                      <path d="M14.348,14.849c-0.469,0.469-1.229,0.469-1.697,0L10,11.819l-2.651,3.029c-0.469,0.469-1.229,0.469-1.697,0
+                               c-0.469-0.469-0.469-1.229,0-1.697l2.758-3.15L5.651,6.849c-0.469-0.469-0.469-1.228,0-1.697s1.228-0.469,1.697,0L10,8.183
+                               l2.651-3.031c0.469-0.469,1.228-0.469,1.697,0s0.469,1.229,0,1.697l-2.758,3.152l2.758,3.15
+                               C14.817,13.62,14.817,14.38,14.348,14.849z" />
+                    </svg>
```

Comment:
nit: use `<i class="fa-solid fa-xmark"></i>`

Reply:
Not applicable anymore, since we're using tom select now